packages/web/src/hooks/useRequiresAccount.ts

```diff
@@ -1,18 +1,6 @@
-import { useCallback, useEffect } from 'react'
+import { useEffect } from 'react'
```

Comment:
I think we can probably just have one implementation of this, so totally down to drop this file. I like useAuthenticatedCallback less than useRequiresAccount just because `auth` can mean a whole lot of things. But it's a digression... this feels great.

I'll test manager mode after this gets merged in

Reply:
Yeah I like requiresAccount better too so I can move everything into this file and do the renames etc